dspublisher/theme/global.css

```
@@ -292,9 +292,22 @@ html[theme~='dark'] .card.card {
   top: -0.1em;
 }
 
-.guilabel {
+[class^='ArticleLayout'] .guilabel {
   font-family: var(--docs-font-family-ui);
   font-size: 0.875em;
   line-height: 1.25;
   font-weight: 600;
 }
+
+[class^='ArticleLayout'] details > summary.title {
+  cursor: pointer;
+  font-size: var(--docs-font-size-m) !important;
+}
+
+[class^='ArticleLayout'] details:not([open]) > summary.title {
+  color: var(--docs-link-color);
+}
+
+[class^='ArticleLayout'] details[open] > summary.title {
+  color: var(--color-secondary-text);
```

Any reason why not use this property instead?

```suggestion
  color: var(--docs-secondary-text-color);
```

---

Ah, didn't notice both existed.